Assets/Scripts/Movement/PlayerMovement.cs

```diff
@@ -41,4 +44,22 @@ private bool isGrounded()
     {
         return Physics2D.Raycast(gameObject.transform.position, Vector3.down, distToGround, LayerMask.GetMask("Ground"));
     }
+
+    private void MirrorPlayer()
+    {
+        float horizontalMovementValue = Input.GetAxis("Horizontal");
+
+        if (horizontalMovementValue < 0)
+        {
+            isFacingRight = false;
+        }
+        else if (horizontalMovementValue > 0)
+        {
+            isFacingRight = true;
+        }
+
+        Vector3 vector = gameObject.transform.localScale;
+        vector.x = isFacingRight ? 1 : -1;
+        gameObject.transform.localScale = vector;
```

Comment:
This localScale assignment gets called every frame. 
To spare unnecessary assignments, create a local variable that will hold the value of isFacingRight before actually calculating the new value. 
Compare the values of old isFacingRight and the new value. Only if they are not equal, execute this 3 lines that change the localScale of the object.

Reply:
done